tests/api/test_alias.py

```diff
@@ -489,6 +490,29 @@ def test_create_contact_route(flask_client):
     assert r.json["existed"]
 
 
+def test_create_contact_route_free_users(flask_client):
+    user = User.create(
```

Comment:
Maybe we can use `get_new_user_and_api_key` here?

Reply:
changed